admin/admin-service/src/test/java/io/mosip/admin/controller/ApplicantDetailsControllerTest.java

```diff
@@ -100,5 +107,15 @@ public void getApplicantDetailsWithInvalidRidTest() throws Exception {
                 (mockMvc.perform(MockMvcRequestBuilders.get("/applicantVerficationDetails"+"/100011019100033202")).andReturn()),
                 "KER-IDV-304");
     }
+    @Test
+    @WithUserDetails(value = "global-admin")
+    public void testGetRIDDigitalCardSuccess() throws Exception {
+        byte[] pdfbytes="test".getBytes();
+        Mockito.when(applicantDetailService.getRIDDigitalCard(Mockito.anyString(),Mockito.anyBoolean())).thenReturn(pdfbytes);
+        mockMvc.perform(MockMvcRequestBuilders.post("/rid-digital-card").param("rid","11234567897").param("isAcknowledged","true")
+                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .andExpect(status().isOk()).andExpect(MockMvcResultMatchers.content()
+                        .contentType(MediaType.APPLICATION_PDF_VALUE));
+    }
```

Comment:
Can we add 2 more test cases, 
1. isAcknowledged is sent as false 
2. Digital card status returns as unavailable

Reply:
taken care

Reply:
Resolved